tests/integ/scala/test_dataframe_reader_suite.py

```diff
@@ -1942,6 +1942,10 @@ def test_read_multiple_csvs(session):
     "config.getoption('local_testing_mode', default=False)",
     reason="xml not supported in local testing mode",
 )
+@pytest.mark.skipif(
+    IS_IN_STORED_PROC,
+    reason="Flaky in stored procedure test",
+)
```

Comment:
FYI @sfc-gh-jdu this is related to xml reader

Reply:
added jira number